R/render-plot.R

```diff
@@ -626,8 +630,8 @@ find_panel_info_api <- function(b) {
 
     # First look for log base in scale, then coord; otherwise NULL.
     list(
-      x = get_log_base(xscale$trans) %OR% coord$xlog %OR% NULL,
-      y = get_log_base(yscale$trans) %OR% coord$ylog %OR% NULL
+      x = get_log_base(xscale$trans) %NA_OR% coord$xlog %NA_OR% NULL,
```

Comment:
`get_log_base()` (defined a few lines above) returns either a number or `NA`. Similarly `coord$xlog` is returned by `ggplot2::summarise_coord()`, and it does the same thing.